src/mlpack/methods/ann/layer/reparametrization_impl.hpp

```diff
@@ -46,7 +46,60 @@ Reparametrization<InputDataType, OutputDataType>::Reparametrization(
         << "included." << std::endl;
   }
 }
+    
+template <typename InputDataType, typename OutputDataType>
+Reparametrization<InputDataType, OutputDataType>::Reparametrization(
+    const Reparametrization& layer) :
+    latentSize(layer.latentSize),
+    stochastic(layer.stochastic),
+    includeKl(layer.includeKl),
+    beta(layer.beta)
+{
+   //Nothing to do here     
+}
 
+template <typename InputDataType, typename OutputDataType>
+Reparametrization<InputDataType, OutputDataType>::Reparametrization(
+    Reparametrization&& layer) :
+    latentSize(std::move(layer.latentSize)),
+    stochastic(std::move(layer.stochastic)),
+    includeKl(std::move(layer.includeKl)),
+    beta(std::move(layer.beta))
+{
+   //Nothing to do here     
+}
+    
+template <typename InputDataType, typename OutputDataType>
+Reparametrization<InputDataType, OutputDataType>&
+Reparametrization<InputDataType, OutputDataType>::
+operator=(const Reparametrization& layer) 
+{
+       if (this != &layer)
+       {
+         latentSize = layer.latentSize;
+         stochastic = layer.stochastic;
+         includeKl = layer.includeKl;
+         beta = layer.beta;
+       }
+       return *this;
+}
+    
+template <typename InputDataType, typename OutputDataType>
+Reparametrization<InputDataType, OutputDataType>&
+Reparametrization<InputDataType, OutputDataType>::
+operator=(Reparametrization&& layer) 
+{
+       if (this != &layer)
+       {
+         latentSize = std::move(layer.latentSize);
+         stochastic = std::move(layer.stochastic);
+         includeKl = std::move(layer.includeKl);
+         beta = std::move(layer.beta);
+       }
+       return *this;
```

Comment:
The indentation is off, should be one space.

Reply:
Thanks for review.. done